docs/api/process.md

```diff
@@ -170,6 +171,18 @@ Returns `Object`:
 
 Returns an object with V8 heap statistics. Note that all statistics are reported in Kilobytes.
 
+### `process.getBlinkMemoryInfo()`
+
+Returns `Object`:
+
+* `allocated` Integer - Total allocated object size in Kilobytes.
+* `marked` Integer - Total marked object size in Kilobytes.
+* `total` Integer - Total allocated space in Kilobytes.
```

Comment:
This probably shouldn't use the word "allocated" because now `allocated` and `total` both say "Total allocated"

Reply:
Or it should just be clarified the difference between this and `allocated`.